cli/src/commands/feature-flag/commands/delete.ts

```diff
@@ -14,6 +14,7 @@ export default (opts: BaseCommandOptions) => {
   command.argument('<name>', 'The name of the feature flag to delete.');
   command.option('-n, --namespace [string]', 'The namespace of the feature flag.');
   command.option('-f --force', 'Flag to force the deletion (skip confirmation).');
+  command.option('--suppress-warnings', 'This flag suppresses the composition warnings.');
```

Comment:
All the places we describe this flag. :D

Reply:
Done